torchvision/io/__init__.py

```diff
@@ -87,15 +91,24 @@ class VideoReader:
 
     def __init__(self, path, stream="video"):
         if not _has_video_opt():
-            raise RuntimeError("Not compiled with video_reader support")
+            raise RuntimeError(
+                "Not compiled with video_reader support, "
+                + "to enable video_reader support, please install "
+                + "ffmpeg (version 4.2 is currently supported) and"
+                + "build torchvision from source."
+            )
         self._c = torch.classes.torchvision.Video(path, stream)
 
     def __next__(self):
-        """Decodes and returns the next frame of the current stream
+        """Decodes and returns the next frame of the current stream.
+        At the moment, frames are encoded as a dict with mandatory
+        data and pts fields, where data is a tensor, and pts is a
+        presentation timestamp of the frame expressed in seconds
+        as a float.
 
         Returns:
             (dict): a dictionary with fields ``data`` and ``pts``
-            containing decoded frame and corresponding timestamp
+                containing decoded frame and corresponding timestamp
```

Comment:
This introduces a bug in rendering the documentation.
Compare previously
![image](https://user-images.githubusercontent.com/9110200/96702691-4c3ab180-1392-11eb-94ea-b2d61ed64bdc.png)
with what we have with this PR:
![image](https://user-images.githubusercontent.com/9110200/96702735-578ddd00-1392-11eb-9bb4-e53851149a9f.png)

Reply:
Should be done in the following commit